vtex-apps/store-components
# Add Title Component

#### What is the purpose of this pull request?
We introduce a title component which can be used to render titles in the store, or in other blocks with center, left and right alignment options.

#### What problem is this solving?
We need to add a title to several `CategoriesHighlights` sections in the homepage. This `Title` component can be added directly to the store's `blocks.json` file or preferably used in the `CategoriesHighlights` component to render the title.

#### How should this be manually tested?
1. Allow `title` block in the `store.home` block in `vtex.store`.
2. Add `title` block to the `store.home` block of the store.
3. Change configuration values in the CMS > Storefront to observe the effects.

#### Screenshots or example usage
![title](https://user-images.githubusercontent.com/42151054/55324138-cea50d00-549e-11e9-923d-51f20a63ae85.png)

#### Types of changes
